From 7ee1c678dcf7310f28b84ff3de46302e271d5757 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Tue, 25 Jun 2019 00:20:53 +0530
Subject: [PATCH 1/2] Remove overriding PhoneSettings package

As Car Settings doesn't have many options needed to verify
features, avoid CarSetttings app overriding Settings app.

Change-Id: I942fa108853e028996e9e9ca558410e32bef402a
Tracked-On:
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 Android.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index fc94a598..f2c3e99f 100644
--- a/Android.mk
+++ b/Android.mk
@@ -22,7 +22,6 @@ include $(CLEAR_VARS)
 # (for example, projected). See b/30064991
 ifeq (,$(TARGET_BUILD_APPS))
   LOCAL_PACKAGE_NAME := CarSettings
-  LOCAL_OVERRIDES_PACKAGES := Settings
   LOCAL_PRIVATE_PLATFORM_APIS := true

   LOCAL_SRC_FILES := $(call all-java-files-under, src)
--
2.21.0
